token-support-tokenx-validation/src/main/kotlin/no/nav/tms/token/support/tokenx/validation/config/HttpClientBuilder.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.tms.token.support.tokenx.validation.config
+
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import io.ktor.client.*
+import io.ktor.client.engine.apache.*
+import io.ktor.client.features.*
+import io.ktor.client.features.json.*
+
+internal object HttpClientBuilder {
+    internal fun build(): HttpClient {
+        return HttpClient(Apache) {
+            install(JsonFeature) {
+                serializer = buildJsonSerializer()
```

Comment:
Her kunne vi vurdert å fjernet Jackson, og heller brukt native-json-serialisering for Kotlin.